app/components/SampleTrees/SampleTreesReview.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { View, Text, FlatList, StyleSheet, Platform, Image } from 'react-native';
+import Label from '../Common/Label';
+import i18next from 'i18next';
+import { Typography } from '_styles';
+import FAIcon from 'react-native-vector-icons/FontAwesome';
+import { Colors } from '_styles';
+import { single_tree_png } from '../../assets';
+import RNFS from 'react-native-fs';
+
+const SampleTreeListItem = ({ sampleTree, index }) => {
+  const imageURIPrefix = Platform.OS === 'android' ? 'file://' : '';
+  let imageSource = sampleTree.imageUrl
+    ? { uri: `${imageURIPrefix}${RNFS.DocumentDirectoryPath}/${sampleTree.imageUrl}` }
+    : single_tree_png;
+
+  return (
+    <View style={styles.specieListItemContainer}>
+      <Image source={imageSource} style={styles.image} resizeMode={'stretch'} />
+      <View style={styles.specieListTextContainer}>
+        <View style={styles.specieHeadingContainer}>
+          <Text style={styles.specieListHeading}>{sampleTree.specieName}</Text>
+          <Text style={styles.specieListHeading}>
+            {sampleTree.tagId ? ' • ' + sampleTree.tagId : ''}
+          </Text>
+        </View>
+        <Text style={styles.subHeadingText}>
+          #{index + 1} • {sampleTree.specieHeight}cm • {sampleTree.specieDiameter}cm
```

Comment:
Please check if we need to have a distinction between cm and inches like done elsewhere depending on the users countryCode.

Reply:
Also height is measured in `m` (or `foot`) while diameter is measured in `cm` (or `inches`).